tests/test_profile.py

```diff
@@ -0,0 +1,35 @@
+from pylexibank.profile import Profile
+
+
+def test_init():
+    prf = Profile({'Grapheme': 'x', 'IPA': 'y'})
+    assert prf.graphemes['^']['IPA'] is None
+
+
+def test_sort():
+    prf = Profile({'Grapheme': 'ab', 'IPA': 'z'}, {'Grapheme': 'x', 'IPA': 'y'})
+    assert list(prf.graphemes.keys())[0] == 'ab'
+    prf.sort()
+    assert list(prf.graphemes.keys())[0] == '^'
+    assert list(prf.graphemes.keys())[-1] == 'ab'
+
+
+def test_trim():
```

Comment:
@tresoldi If you have ideas for more tests, let me know!

Reply:
I could add a couple, if you prefer. The most important thing is to check if it is handling all cases with slash notation correctly (I think there might be a bug in my code for cases where the left side is empty, for example).

Alternatively, I can just work on one more profile and will probably hit all cases that would deserve testing. :wink: